packages/presentation-exchange/src/agent/PresentationExchange.ts

```diff
@@ -184,24 +188,8 @@ export class PresentationExchange implements IAgentPlugin {
   }
 
   /*private assertIdentifier(identifier?: IIdentifier): void {
-    if (!identifier) {
-      throw Error(`OID4VP needs an identifier at this point`)
-    }
-  }*/
-
-  private store({ storeId }: { storeId?: string }): IKeyValueStore<IPresentationDefinition> {
-    const store = this._stores.get(storeId ?? this.defaultStore)
-    if (!store) {
-      throw Error(`Could not get definition store: ${storeId ?? this.defaultStore}`)
-    }
-    return store
-  }
-
-  private namespace({ namespace }: { namespace?: string }): string {
-    return namespace ?? this.defaultStore
-  }
-
-  private prefix({ namespace, definitionId }: { namespace?: string; definitionId: string }) {
-    return `${this.namespace({ namespace })}:${definitionId}`
-  }
+                if (!identifier) {
```

Comment:
this can be removed?

Reply:
Nobody missed it the last 12 months, so deleted now